crates/miden-lib/asm/kernels/transaction/lib/faucet.masm

```diff
@@ -192,7 +191,7 @@ end
 #!   transaction via a note or the accounts vault.
 proc.burn_non_fungible_asset
     # assert that we are executing a transaction against the non-fungible faucet (access checks)
-    exec.account::get_id exec.account::is_non_fungible_faucet 
+    exec.account::get_id swap drop exec.account::is_non_fungible_faucet 
```

Comment:
I think I don't understand why this worked before :thinking:.

Reply:
Correct, it wasn't working at all, and we didn't notice it because of the ignored tests

Reply:
Mh, wouldn't this mean we can't issue non-fungible assets on the testnet?

Reply:
I'm not sure how does it work on the testnet, but looks like for now minting and burning of the non-fungible assets is unavailable, yes.